### ISSUE TITLE
Badges plugin

### ISSUE THREAD

**Issue by boonebgorges** [cuny-academic-commons/commons-in-a-box#203]
The OpenLab has a custom Badges plugin that they'd like to include in the CBOX-OL package. Tasks:

- [x] Check i18n
- [x] Do security review
- [x] Check for OpenLab and City Tech hardcoded strings
- [x] Break into its own repo (it's already its own plugin on the OpenLab, but is not appropriate for the wordpress.org repo, so it should be GitHub-only)
- [x] Make changes described in OL tickets: http://redmine.citytech.cuny.edu/issues/2739, http://redmine.citytech.cuny.edu/issues/2738
- [x] Add to the `openlab` plugin manifest

**Comment by boonebgorges** [comment]
New repo created at https://github.com/livinglab/openlab-badges

**Comment by boonebgorges** [comment]
I've done a review for security and i18n.

**Comment by boonebgorges** [comment]
I've finished most of the work to implement the new badge layout, as well as the new badge filters in group directories. I think that the first round of testing is likely to take place on the OpenLab in the first or second week of June, and there may be additional changes required here based on those changes. I'll hold off on tagging the plugin - and including it in the `openlab` manifest - until that time.

**Comment by boonebgorges** [comment]
I've been working on integration for Badges and I realized that the Badge admin panel, in the Network Admin, might be difficult to find for some network administrators. As such, I suggest a subtab under 'Group Settings' that points users to this admin page, similar to what we do under Communication Settings for other plugins. See screenshot. @bree-z @jennaspevack Are you OK with this approach?

![Screenshot_2020-07-24_15-32-08](https://user-images.githubusercontent.com/246627/88408378-00906000-cdc3-11ea-9acc-5ad9f34a98f1.png)

**Comment by boonebgorges** [comment]
I've taken a first pass at integration. It basically works in the same way as the OpenLab, with the following changes:

1. I've added the Group Settings > Badges admin tab, as in the screenshot above
2. The 'Manage Badges' button does not appear under the avatar on group home. We removed all the admin buttons as part of cuny-academic-commons/openlab-theme#135, so I didn't add this one either.
3. The single-group settings appear under Settings > Settings (`/admin/group-settings/`)
